sdks/java/io/cassandra/build.gradle

```diff
@@ -17,7 +17,24 @@
  */
 
 apply plugin: org.apache.beam.gradle.BeamModulePlugin
-applyJavaNature()
+applyJavaNature(shadowClosure: {
```

Comment:
I've actually gone in the opposite direction and replaced this with a blanket "exclude everything", since guava is now provided by the beam vendored version (except where it needs to use `ListenableFuture` from the driver, which comes from regular guava).
This seems acceptable to me because the default rule is "include guava" (and exclude everything else) but let me know if you disagree.

Reply:
This seems like the most restricted (and I would say proper way), my doubt is if there are some some other uses of Cassandra's guava that we could be missing and that can bite us later.

Reply:
Will this do both?

1. Don't relocate anything
2. Don't bundle anything

I think that is what you want, now. And since the vendored Guava types are incompatible with non-vendored Guava types, it is impossible to mix them up.

Reply:
This is the best approach but we probably need to document in the IO that this may leak guava 19 (from Cassandra's driver) and this can have unintended consequences no? at least until all Beam's base modules use the vendored version as a dependency too. or am I misreading @kennknowles?

Side discussion (not related to the Cassandra case): I was taking a look at the use of vendored versions and could not find many docs/examples I suppose that this is because this is in progress? Anyway I can help there. I just found that many core modules `beam-runners-core-java, beam-runners-core-construction-java, beam-model-fn-execution, etc.` are still repackaging guava its own way, but more critical that `beam-runners-direct-java` repackages guava 5 times because of its dependencies which is particularly bad.